web-core-sdk/algorand/react-algorand-core-example/src/algorandRPC.ts

```diff
@@ -0,0 +1,88 @@
+import { SafeEventEmitterProvider } from "@web3auth/base";
+import algosdk from "algosdk";
+
+export default class AlgorandRPC {
+  private provider: SafeEventEmitterProvider;
+
+  constructor(provider: SafeEventEmitterProvider) {
+    this.provider = provider;
+  }
+
+  getAlgorandKeyPair = async (): Promise<any> => {
+    const privateKey = (await this.provider.request({
+      method: "private_key",
+    })) as string;
+    var passphrase = algosdk.secretKeyToMnemonic(
+      Buffer.from(privateKey, "hex")
+    );
+    var account = algosdk.mnemonicToSecretKey(passphrase);
```

Comment:
This is `key-pair`, we shouldn’t write `account` for this. Please change everywhere too.

account is in `obj.addr` right?

Reply:
Yeah, made the changes as per requested. The public key is ```obj.addr``` .